app/views/datasets/_meta_data.html.erb

```diff
@@ -13,15 +12,15 @@
             <dt>Last updated:</dt>
             <dd><%= format(@dataset["updated_at"]) %></dd>
             <dt>Expected update:</dt>
-              <% if @dataset["datafiles"].any? %>
-                <dd class="<%= expected_update_class_for(@dataset['frequency'])%>">
+            <% if @dataset["datafiles"].any? %>
+                <dd class="<%= expected_update_class_for(@dataset['frequency']) %>">
```

Comment:
mmm ... this appears to be default settings via the Intellij ruby plugin
Its two spaces but this increases to four when you start a new line after executing ruby code <% ... %>. This can be changed though I'm sure. 
Let's discuss